test/gateway/polygon-pos.test.ts

```diff
@@ -11,9 +11,6 @@ const config = PolygonPoSRollup.mainnetConfig;
 // 20240923: disabled until polygon has non-erigon rpcs
```

Comment:
These still consistently fail for me. I think as a side point its probably worth commenting in tests etc the constraints of the current setups. Will discuss further with you..

Reply:
![image](https://github.com/user-attachments/assets/882a50c0-851d-43ce-a2c8-bc7df3efe935)

Ya, its basically unusable.